resources/label-definitions.properties

```diff
@@ -439,7 +439,7 @@ housekeeper=post-build
 hp-application-automation-tools-plugin=misc
 hp-operations-orchestration-automation-execution-plugin=misc external
 hp-quality-center=post-build
-hpe-network-virtualization=misc
+hpe-network-virtualization=deprecated
```

Comment:
This is unnecessary and unused if the plugin is suspended.

Reply:
Would you like me to roll back this change?

Reply:
Ideally, given the removal of the plugin,
```suggestion
```